### DISCUSSION TITLE
problem

### DISCUSSION THREAD

**Discussion by Aiozlove** [winstxnhdw/lc-hax#152] [Q&A]
Do you have a compilation tutorial because I am a fool.

**Comment by winstxnhdw** [comment]
No, there isn't, the README is all we have. But don't worry, there's dozens of such cases if you look around `Discussions`.

**Comment by Aiozlove** [comment]
Can you provide a tutorial?which is very helpful for beginners.

**Reply by winstxnhdw** [reply]
Why not look at the README?
